src/asset/asset.js

```diff
@@ -61,6 +61,7 @@ Object.assign(pc, function () {
      * var asset = new pc.Asset("a texture", "texture", {
      *     url: "http://example.com/my/assets/here/texture.png"
      * });
+     * @param {object} [options] - a JSON object containing load-time options specific to the asset.
```

Comment:
You don't document the options. 😄

Reply:
Yeah that's an interesting one. The options structure will depend on the asset type. For textures so far it's one crossOrigin member. Do you have any ideas how we go about documenting that?

Reply:
List the options and in brackets put `(only applies to assets of type 'texture')`?